### PR TITLE
DEVSVCS-592 [P0] Secrets encrypt CLI - Add encryptionPublicKey

### PR DESCRIPTION
**Pull request by KuphJr**
[smartcontract-it.atlassian.net/browse/DEVSVCS-592](https://smartcontract-it.atlassian.net/browse/DEVSVCS-592)
For ST3, each node is to have its own public key so that users can encrypt secrets for that particular node.

### CODE REVIEW

File: integration-tests/deployment/ccip/deploy_home_chain.go [hunk 1]
```diff
@@ -206,8 +206,9 @@ func AddNodes(
 		}
 		nodeParam := capabilities_registry.CapabilitiesRegistryNodeParams{
 			NodeOperatorId:      NodeOperatorID,
-			Signer:              p2pID, // Not used in tests
```
Comment (DeividasK) [inline integration-tests/deployment/ccip/deploy_home_chain.go:209]: Was this an intentional removal?

**Review by DeividasK — DISMISSED** [review]
LGTM

File: core/scripts/keystone/src/05_deploy_initialize_capabilities_registry.go [hunk 2]
```diff
@@ -26,110 +26,134 @@ import (
...
 }
 
 var (
 	workflowDonPeers = []peer{
 		{
-			PeerID: "12D3KooWBCF1XT5Wi8FzfgNCqRL76Swv8TRU3TiD4QiJm8NMNX7N",
-			Signer: "0x9639dCc7D0ca4468B5f684ef89F12F0B365c9F6d",
+			PeerID:              "12D3KooWBCF1XT5Wi8FzfgNCqRL76Swv8TRU3TiD4QiJm8NMNX7N",
+			Signer:              "0x9639dCc7D0ca4468B5f684ef89F12F0B365c9F6d",
+			EncryptionPublicKey: "0xe7f44e3eedf3527199efec7334183b5384ba0e7c7c57b390b63a3de5a10cd53c",
```
Comment (cedric-cordenier) [inline core/scripts/keystone/src/05_deploy_initialize_capabilities_registry.go:39]: Thanks! Do we have the private keys stored somewhere?

**Review by cedric-cordenier — DISMISSED** [review]

File: core/scripts/keystone/src/05_deploy_initialize_capabilities_registry.go (hunk 2 above)
Comment (KuphJr) [reply core/scripts/keystone/src/05_deploy_initialize_capabilities_registry.go:39]: These are random values.

File: integration-tests/deployment/ccip/deploy_home_chain.go [hunk 3]
```diff
@@ -206,8 +206,9 @@ func AddNodes(
...
+			Signer:              p2pID,
```
Comment (KuphJr) [inline integration-tests/deployment/ccip/deploy_home_chain.go]:
```suggestion
			Signer:              p2pID, // Not used in tests
```

**Review by cedric-cordenier — DISMISSED** [review]

**Review by DeividasK — DISMISSED** [review]

**Review by cedric-cordenier — APPROVED** [review]

**Review by shileiwill — APPROVED** [review]
LGTM!

**Review by HouseOfHufflepuff — APPROVED** [review]

**Review by AndresJulia — APPROVED** [review]

**Review by winder — APPROVED** [review]

**Review by rstout — APPROVED** [review]

**Review by RensR — APPROVED** [review]

**Pull request merged**